pyboy/core/cartridge/mbc1.py

```diff
@@ -11,28 +11,14 @@
 class MBC1(BaseMBC):
     def setitem(self, address, value):
         if 0x0000 <= address < 0x2000:
-            if (value & 0b00001111) == 0x0A:
-                self.rambank_enabled = True
-            else:
-                self.rambank_enabled = False
+            self.rambank_enabled = (value & 0b00001111) == 0b1010
         elif 0x2000 <= address < 0x4000:
-            # But (when using the register below to specify the upper ROM Bank bits), the same
-            # happens for Bank 20h, 40h, and 60h. Any attempt to address these ROM Banks will
-            # select Bank 21h, 41h, and 61h instead.
+            value &= 0x1F
             if value == 0:
                 value = 1
-            # sets 5LSB of ROM bank address
-            self.rombank_selected = self.rombank_selected & 0b11100000 | value & 0b00011111
+            self.bank1 = value
```

Comment:
Wouldn't it have worked with the old code? It seems like the variables `self.bank1` and `self.bank2` could be avoided?